lms/static/js/instructor_dashboard/data_download.js

```diff
@@ -118,7 +118,30 @@
             this.instructor_tasks = new (PendingInstructorTasks())(this.$section);
             this.clear_display();
             this.$list_anon_btn.click(function() {
-                location.href = dataDownloadObj.$list_anon_btn.data('endpoint');
+                var url = dataDownloadObj.$list_anon_btn.data('endpoint');
+                var errorMessage = gettext('Error generating anon ids results. Please try again.');
```

Comment:
Nit: `Error generating anonymous IDs. Please try again.` I feel this is easier to understand for translators